libs/eo/landing-page/shell/src/lib/layout/eo-landing-page-notification.component.ts

```diff
@@ -33,50 +35,24 @@ const selector = 'eo-landing-page-notification';
       ${selector} {
         display: block;
       }
-
-      .${selector}__content {
-        display: grid;
-        grid-template-columns: 80px 1fr;
-        align-items: center;
-        background: var(--watt-color-neutral-white);
+      .${selector}__eo-inline-message {
         margin: var(--watt-space-m) auto;
         width: calc(259 * var(--watt-space-xs));
-        box-shadow: 0px 2px 5px rgba(0, 0, 0, 0.08);
-      }
-
-      .${selector}__icon {
-        display: block;
-        width: calc(12 * var(--watt-space-xs));
-        height: calc(12 * var(--watt-space-xs));
-        border-radius: 50%;
-        margin: 0 auto;
-      }
-
-      .${selector}__p {
-        @include watt.typography-watt-text-m; // This overrides the styles applied from Angular Material on p tags
-        padding-right: calc(4 * var(--watt-space-xs));
-        color: var(--watt-color-primary-dark);
       }
     `,
   ],
   template: `
-    <div class="${selector}__content">
-      <div>
-        <img
-          class="${selector}__icon"
-          src="/assets/images/icons/primary-info-icon.svg"
-          alt="EnergyOrigin information"
-        />
-      </div>
-      <div>
-        <p class="${selector}__p">
-          The Energy Origin Platform is <b>under development</b> and new
-          functionalities will be released continuously. The first release of
-          the platform offers <b>company login only</b>. Private login via
-          NemID/MitID is intended to form part of one of the next releases.
-        </p>
-      </div>
-    </div>
+    <eo-inline-message
+      icon="/assets/images/icons/primary-info-icon.svg"
+      class="${selector}__eo-inline-message"
+    >
+      <p>
+        The Energy Origin Platform is <b>under development</b> and new
```

Comment:
issue: Don't use the `<b>` tag, it was deprecated 14 years ago 🙂 Look for uses in other pages.
```suggestion
        The Energy Origin Platform is <strong>under development</strong> and new
```